notebooks/community/vertex_endpoints/optimized_tensorflow_runtime/bert_optimized_online_prediction.ipynb

```diff
@@ -1510,7 +1510,7 @@
       "source": [
```

Comment:
<p>Can you replace GAPIC calls with SDK calls.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/630/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='81'/>

Reply:
<p>That's a major refactoring, let me do that in a follow-up PR.</p>